crates/release_plz_core/src/command/release.rs

```diff
@@ -961,54 +1119,6 @@ async fn release_package(
             wait_until_published(index, release_info.package, input.publish_timeout, token).await?;
         }
 
-        if should_create_git_tag {
```

Comment:
The `should_create_git_tag` and `should_create_git_release` variables are now unused in this function, only debug-logged; that debug log should probably moved to the function where these variables are actually used, and the duplicate definition removed.